src/org/loklak/tools/NetworkIO.java

```diff
@@ -35,77 +35,110 @@
+    private final static String HEADER_PARAM_CONTENT_TYPE = "Content-Type: text/plain; charset=";
 
+    /**
+     *Sends GET request to host API.
+     * @param url Url for GET request
+     * @return Server reply as String (in JSON format) from host API
+     * @throws IOException
+     */
     public static StringBuilder loadString(String url) throws IOException {
         StringBuilder sb = new StringBuilder();
         URLConnection uc = (new URL(url)).openConnection();
-        HttpURLConnection con = url.startsWith("https") ? (HttpsURLConnection) uc : (HttpURLConnection) uc;
+        HttpURLConnection connection =
+                url.startsWith("https") ? (HttpsURLConnection) uc : (HttpURLConnection) uc;
         try {
-            con.setReadTimeout(6000);
-            con.setConnectTimeout(6000);
-            con.setRequestMethod("GET");
-            con.setDoInput(true);
-            con.setRequestProperty("User-Agent", USER_AGENT);
-            sb = load(con);
+            connection.setReadTimeout(6000);
+            connection.setConnectTimeout(6000);
+            connection.setRequestMethod(REQUEST_METHOD);
+            connection.setDoInput(true);
+            connection.setRequestProperty(REQUEST_PROPERTY_USER_AGENT, USER_AGENT);
+            sb = load(connection);
         } catch (IOException e) {
             throw e;
         } finally {
-            con.disconnect();
-            
+            connection.disconnect();
         }
         return sb;
     }
 
-    private static StringBuilder load(HttpURLConnection con) throws IOException {
+    private static StringBuilder load(HttpURLConnection connection) throws IOException {
         StringBuilder sb = new StringBuilder(1024);
         BufferedReader br = null;
         try {
-            br = new BufferedReader(new InputStreamReader(con.getInputStream(), "UTF-8"));
+            br = new BufferedReader(new InputStreamReader(connection.getInputStream(), CHARSET));
             String s;
-            while ((s = br.readLine()) != null) sb.append(s).append('\n');
+            while ((s = br.readLine()) != null) {
+                sb.append(s).append('\n');
+            }
         } catch (IOException e) {
             throw e;
         } finally {
             if (br != null) br.close();
-            con.disconnect();
+            connection.disconnect();
         }
         return sb;
     }
-    
-    public static StringBuilder pushString(String requestURL, String jsonDataName, String bodytext) throws IOException, JSONException {
+
+    /**
+     *Sends POST request to host API.
+     * @param requestURL Url for POST request
+     * @param jsonDataKey Name of JSONObject's key
+     * @param bodyText JSONObject as String
+     * @return Server reply as String (in JSON format) from host API
+     * @throws IOException
+     * @throws JSONException
+     */
+    public static StringBuilder pushString(String requestURL, String jsonDataKey, String bodyText)
```

Comment:
Looking at the rest of your code where you put them into multiple lines, this becomes inconsistent. It will be better if all these functions look exactly like this, one liners with exceptions thrown on next line or even better if on the same line.